OSDOCS#12641: Reorganize and add docs for etcd recovery (#88498)
openshift/openshift-docs: modules/dr-restoring-cluster-state-sno.adoc

// Module included in the following assemblies:
//
// * disaster_recovery/scenario-2-restoring-cluster-state.adoc

:_mod-docs-content-type: PROCEDURE
[id="dr-restoring-cluster-state-sno_{context}"]
= Restoring to a previous cluster state for a single node

> Follow these steps to restore your cluster to a previous state for {product-title} on a single node.

Reviewer: Possible to rephrase as a use case instead of being self-referential?

[IMPORTANT]
====

> When you restore your cluster, you must use an `etcd` backup that was taken from the same z-stream release. For example, an {product-title} 4.7.2 cluster must use an `etcd` backup that was taken from 4.7.2.

Reviewer: Should etcd be in backticks here? Usually it's not
```suggestion
When you restore your cluster, you must use an `etcd` backup that was taken from the same z-stream release. For example, an {product-title} {product-version}.2 cluster must use an `etcd` backup that was taken from {product-version}.2.
```

====

.Prerequisites

* Access to the cluster as a user with the `cluster-admin` role through a certificate-based `kubeconfig` file, like the one that was used during installation.
* SSH access to control plane hosts.
* A backup directory containing both the `etcd` snapshot and the resources for the static pods, which were from the same backup. The file names in the directory must be in the following formats: `snapshot_<datetimestamp>.db` and `static_kuberesources_<datetimestamp>.tar.gz`.

.Procedure

// Does the `cp` command require `sudo`?
. SSH into the single node and copy the etcd backup to the `/home/core` directory:
+
[source,terminal]

> ====

Reviewer: need to replace `====` with `----` in this proc too :)

> $ cp <etcd-backup-directory> /home/core

Reviewer: ```suggestion
$ cp <etcd_backup_directory> /home/core
```

====

> . Run the following command in the single node:

Reviewer: Describe what this does?

+
[source,terminal]
====

> $ sudo -E /usr/local/bin/cluster-restore.sh /home/core/<etcd-backup-directory>

Reviewer: ```suggestion
$ sudo -E /usr/local/bin/cluster-restore.sh /home/core/<etcd_backup_directory>
```

====

> . Exit the SSH session and wait until the control plane recovers by running:

Reviewer: ```suggestion
. Exit the SSH session.

. Monitor the the control plane recovery progress by running the following command:
```

// Module included in the following assemblies:
//
// * disaster_recovery/scenario-2-restoring-cluster-state.adoc

:_mod-docs-content-type: PROCEDURE
[id="dr-restoring-cluster-state-sno_{context}"]
= Restoring to a previous cluster state for a single node

You can use a saved etcd backup to restore a previous cluster state on a single node.

[IMPORTANT]
====
When you restore your cluster, you must use an etcd backup that was taken from the same z-stream release. For example, an {product-title} {product-version}.2 cluster must use an etcd backup that was taken from {product-version}.2.
====

.Prerequisites

* Access to the cluster as a user with the `cluster-admin` role through a certificate-based `kubeconfig` file, like the one that was used during installation.
* You have SSH access to control plane hosts.
* A backup directory containing both the etcd snapshot and the resources for the static pods, which were from the same backup. The file names in the directory must be in the following formats: `snapshot_<datetimestamp>.db` and `static_kuberesources_<datetimestamp>.tar.gz`.

.Procedure

> . Use SSH to connect to the single node and copy the etcd backup to the `/home/core` directory:

Reviewer 2: ```suggestion
. Use SSH to connect to the single node and copy the etcd backup to the `/home/core` directory by running the following command:
```
